frontend/src/views/Main.vue

```diff
@@ -76,12 +76,27 @@
             dark
             color="green lighten-1"
             v-on="on"
-            @click="createAbfrage"
+            @click="createBauleitplanverfahren"
```

Comment:
Wäre es vom Handling nicht besser anstelle der einzelnen Button für das Baugenehmigungsverfahren und Bauleitplanverfharen einen Speeddial für Abfragen zu machen welcher dann die Button für  die zwei bzw. später drei Abfragen aufklappt? Als einen Speedial innerhalb des jetzigen Speeddials.

Reply:
Ich habe es versucht und dabei leider nicht in einem vertretbaren Zeitrahmen hinbekommen.
Wenn ich am Sprint Ende noch Luft habe, werde ich mich dem Thema gerne nochmal widmen.